Enterprise_Dbcache_WpdbInjection_Cluster.php

```diff
@@ -559,7 +561,7 @@ function query( $query ) {
 		$this->wpdb_mixin->last_query = $query;
 
 		if ( preg_match( '/^\s*SELECT\s+FOUND_ROWS(\s*)/i', $query )
-			&& is_resource( $this->wpdb_mixin->_last_found_rows_result ) ) {
+			&& is_object( $this->wpdb_mixin->_last_found_rows_result ) ) {
```

Comment:
This should work.  A stricter check would be to use `is_a( $this->wpdb_mixin->_last_found_rows_result, 'mysqli_result' )`, although `is_object()` is ok.